### ISSUE TITLE
[BUG] RemoteTech booting for new Vessels or Sceene Switch

### ISSUE THREAD

**Issue by GER-Space** [KSP-KOS/KOS#1775]
I have to ressurect the Bug #453 
When testing Kerbal-EVA I run always in the issue that kOS initializes faster that RemoteTech and refuses to boot. I also see it sometimes when switching to a vessel, which is out of range with KUNIVERSE:ACTIVEVESSEL or when switching from the from KSC.

I have no Idea how to delay the RemoteTechArchive.CheckRange check. All my experiments resulted in a failure :-(

**Comment by hvacengi** [comment]
The best answer I can recommend for the time being is to not boot on the archive while using Remote Tech.  You don't even have to do all of your actual work on the local drive, but at least make it so that your boot file is local, and then it can wait for the RT connection before switching to the archive.

```
if addons:available("RT") { wait until haskscconnection(ship). }
switch to 1.
```

The alternative is that you will have to reboot the cpu (which should now work).  Realistically it is the same solution you would use if you legitimately booted with no KSC connection.

There is no good solution that I am aware of for us to determine that RemoteTech has finished loading the network.

**Comment by hvacengi** [comment]
I seem to think this issue should have been fixed as part of #1850.  In that pull request we no longer have an error if the archive is not in range on boot, but instead just throw an execution error whenever you try to actually do something on the archive.  I will close this issue for the time being, please re-open it if you continue to have issues.
